hxl_proxy/controllers.py

```diff
@@ -40,11 +42,15 @@ def handle_default_exception(e):
     @param e: the exception being handled
     """
 
+    key_error = {'error_type': type(e).__name__}
     # log a warning for the error/exception that we're handling
     if hasattr(e, 'message'):
+        key_error['message'] = e.message
         logger.warning('%s: %s', type(e).__name__, e.message)
     else:
         logger.warning(type(e).__name__)
+    logup('Error', key_error, "warning")
```

Comment:
Do we need to keep logger.warning/logger.info _and_ logup throughout, or would just logup be OK?

Reply:
I didn't want to start removing lines in the code.
I just added the json logs.
Tbh I was convinced these `logger.warning` are going to be displayed to the client in the html page at some point.

Reply:
OK, that's fair.